src/open_inwoner/configurations/models.py

```diff
@@ -249,6 +250,15 @@ class SiteConfiguration(SingletonModel):
         default="",
         help_text=_("The external link for the footer logo."),
     )
+    favicon = FilerImageField(
+        verbose_name=_("Favicon image (16x16)"),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="favicon",
+        help_text=_("Image to use as favicon"),
+        validators=[FilerExactImageSizeValidator(16, 16)],
```

Comment:
32x32? https://stackoverflow.com/questions/4014823/does-a-favicon-have-to-be-32%C3%9732-or-16%C3%9716

Reply:
@alextreme Sure, I added a commit to update this to 32x32